bindings/rust/s2n-tls/src/client_hello.rs

```diff
@@ -0,0 +1,388 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+use std::fmt;
```

Comment:
Nit: why are all of the imports on separate lines?

Reply:
huh, til about [import groupings](https://rust-lang.github.io/rustfmt/?version=v1.4.38&search=#group_imports)

I had assumed that rustfmt would fix that for me so I never paid much attention to that.